packages/repo-doctor/src/reporters/mattermost.js

```diff
@@ -0,0 +1,55 @@
+const fetch = require('node-fetch')
+
+const symbolBySeverity = {
+  warn: '⚠️',
+  success: '✅',
+  error: '🚩',
+  info: 'ℹ️'
+}
+
+class MattermostReporter {
+  constructor(config) {
+    this.config = config
+    this.checkEnv()
+    this.messages = []
+  }
+
+  checkEnv() {
+    if (!process.env.MATTERMOST_HOOK) {
```

Comment:
Wouldn't it make sense to have this variable as part of the configuration rather than an env variable?

Reply:
I'm not sure about that since the hook is supposed to be secret. I don't think we want this to be written in our config file.